src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -389,10 +345,18 @@ class HoldRequest extends React.Component {
     }
 
     const searchUrl = basicQuery(this.props)({});
+    const userLoggedIn = this.props.patron && this.props.patron.loggedIn;
```

Comment:
we're going to have a discussion about auth but this looks like it is working well for now 👍